src/test/java/org/intellij/sdk/codesync/UtilsTest.kt

```diff
@@ -0,0 +1,68 @@
+package org.intellij.sdk.codesync
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import org.intellij.sdk.codesync.utils.ProjectUtils
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.*
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class UtilsTest {
```

Comment:
If getAllContentRoots isn't the correct fix, Why our unit tests catch the issue? @syedgulahmed